tasks/main.yml

```diff
@@ -28,5 +28,13 @@
   notify:
     - Restart Zookeeper
 
+- name: Configure JAVA_OPTS
```

Comment:
I was unable to get this addition to work until I set `zookeeper_java_opts` to:

```yaml
zookeeper_java_opts: "export SERVER_JVMFLAGS=-Xmx2048m"
```

Through the process of figuring that out, I noticed that the `/etc/default/zookeeper` file contains a commented out reference to `SERVER_JVMFLAGS`. Does it make more sense to just template out vs. creating `java.env`?